website/docs/d/oracle_db_nodes.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_db_nodes"
+description: |-
+  This data source provides the list of Db Nodes.
+---
+
+# Data Source: azurerm_oracle_db_nodes
+
+Lists the database nodes for the specified Cloud VM Cluster.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_db_nodes" "example" {
+  resource_group_name   = "existing"
+  cloud_vm_cluster_name = "existing"
+}
+
+output "id" {
+  value = data.azurerm_oracle_db_nodes.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `cloud_vm_cluster_name` - (Required) The name of the Cloud VM cluster.
+
+* `resource_group_name` - (Required) The name of the Resource Group the Cloud VM Cluster.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `db_nodes` - A `db_nodes` block as defined below.
+
+---
+
+A `db_nodes` block exports the following:
+
+* `additional_details` - Additional information about the planned maintenance.
+
+* `backup_ip_id` - The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the backup IP address associated with the database node. Use this OCID with either the [GetPrivateIp](https://docs.cloud.oracle.com/iaas/api/#/en/iaas/20160918/PrivateIp/GetPrivateIp) or the [GetPublicIpByPrivateIpId](https://docs.cloud.oracle.com/iaas/api/#/en/iaas/20160918/PublicIp/GetPublicIpByPrivateIpId) API to get the IP address needed to make a database connection.
+
+* `backup_vnic_id` - The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the backup VNIC.
+
+* `backup_vnic2id` - The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the second backup VNIC.
+
+* `cpu_core_count` - The number of CPU cores enabled on the Db node.
```

Comment:
Please update all references of `Db` -> `DB` in this doc

Reply:
Done, thanks.